packages/editor/src/components/post-publish-button/index.js

```diff
@@ -155,7 +155,7 @@ export default compose( [
 	withDispatch( ( dispatch ) => {
 		const { editPost, savePost } = dispatch( 'core/editor' );
 		return {
-			onStatusChange: ( status ) => editPost( { status } ),
+			onStatusChange: ( status ) => editPost( { status }, { undoIgnore: true } ),
```

Comment:
I wonder if this is the right place to make this decision. Should status changes in general be ignored? Or publish status changes? Genuinely wondering...

Reply:
It depends on the context and the entity.

This will cover publishing status changes for posts and it makes sense to ignore that.

It will be very context-dependent which is why I didn't make it part of the entity configuration.

Reply:
Ok fair enough.